### PR TITLE
Fix log format when no log file is specified

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

When no logFile is specified an extra parameter (logFileMaxSize) would
be given to the final Sprintf and used as the date format.

This would procude an output such as:
```
%!s(int=507859760) | INFO | (log.go:266 in Info) | starting the tagging system
%!s(int=507859760) | INFO | (log.go:218 in Infof) | docker tag collector successfully started
%!s(int=50880760) | INFO | (log.go:218 in Infof) | Runner started with 1 workers.
```

### CODE REVIEW

File: releasenotes/notes/fix-check-cmd-log-format-55837ac1c66ef62a.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fix log format when no log file would be specified which cause the log date
```
Comment (olivielpeau) [inline releasenotes/notes/fix-check-cmd-log-format-55837ac1c66ef62a.yaml]: `when no log file is specified`

File: pkg/config/log.go [hunk 2]
```diff
@@ -86,7 +84,7 @@ func SetupLogger(logLevel, logFile, uri string, rfc, tls bool, pem string, logTo
 
 	configTemplate += `</formats>
 	</seelog>`
-	config := fmt.Sprintf(configTemplate, seelogLogLevel, logFile, logFileMaxSize, logDateFormat, logDateFormat)
+	config := fmt.Sprintf(configTemplate, seelogLogLevel, logDateFormat, logDateFormat)
```
Comment (olivielpeau) [inline pkg/config/log.go]: additional improvement we could make here, if you have time: formatting these last 3 items earlier, currently it's very hard to understand where in the `configTemplate` string these are supposed to be used.

Feel free to merge without addressing this though :) We could plan work on refactoring this whole function, which is becoming a bit of a mess.

**Review by olivielpeau — DISMISSED** [review]
Thanks for fixing this! One nit on the release note, and another comment (feel free not to address it though)

File: pkg/config/log.go (hunk 2 above)
Comment (hush-hush) [reply pkg/config/log.go]: Done !

**Review by olivielpeau — APPROVED** [review]
👍

**Pull request merged**